src/main/java/org/fenixedu/academictreasury/dto/reports/DebtReportEntryBean.java

```diff
@@ -546,7 +554,26 @@ public void writeCellValues(final Row row, final IErrorsLog ierrorsLog) {
                     }
                     row.createCell(i++).setCellValue(valueOrEmpty(value));
                 }
-                
+
+                {
+                    String value = openAmountWithInterestToDate != null ? openAmountWithInterestToDate.toString() : "";
+                    if (DebtReportRequest.COMMA.equals(decimalSeparator)) {
+                        value = value.replace(DebtReportRequest.DOT, DebtReportRequest.COMMA);
+                    }
+                    row.createCell(i++).setCellValue(valueOrEmpty(value));
+                }
+
+                if (TreasurySettings.getInstance().getInterestProduct().getCode().equals(productCode)) {
```

Comment:
Hello.

One debt of "Interest" product might have pending interest amount if it is configured to with a tax rate. You can print with zero amount like other entries without interests tax rate configured.

Just a tip, you can check if it is the debt is of Interest product comparing the object references. In this case:

`TreasurySettings.getInstance().getInterestProduct() == invoiceEntry.getProduct()`

Reply:
Hello, as requested, I pushed a change to consider that observation.